tests/performance/run-locust.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# Example usage:
+#
+# $ docker build -t amoloadtests:latest -f Dockerfile.perftests
+# $ docker run -ti -e LOCUST_OPTS="-c 1 --no-web" \
+#   -e ATTACKED_HOST="https://addons.allizom.org" \
+#   amoloadtests:latest
+#   /code/tests/performance/run-locust.sh
+#
+# If you're running this against addons.allizom.org make sure to configure
+# the correct FxA environment variables:
+#
+# -e FXA_CLIENT_ID="..." -e FXA_CLIENT_SECRET="..."
```

Comment:
How would one get these details? Copy them from somewhere else or did you have to generate a new oauth client for it?

Reply:
I haven't, I used the local fxa credentials for now and once this is deployed and on stage we can just run it from there or a special machine and have the environment variables ready.